Add Python version/implementation classifiers.

jamadden opened the pull request:
This is necessary for tools like @mgedmin 's site https://zope3.pov.lt/py3/ and caniusepython3.com to function correctly.

Also add setup.cfg to indicate that wheels are universal.

jamadden committed:
Add Python version classifiers.

Also add setup.cfg to indicate that wheels are universal.

Fix tests on Travis and enable coveralls.

Add appropriate .coveragerc.

mgedmin commented:
Tests fail because .travis.yml is bad -- we should be using python: [ 2.7, 3.5, 3.6, pypy ] and tox-travis.

jamadden commented:
Or avoid tox altogether and take advantage of pip caching.

mgedmin commented:
tox should be able to take advantage of pip caching, AFAIU -- unless the pip/virtualenv versions in the Travis image are too old?

(The best reason I know not to use tox is when you want to use coverage run + coveralls.)

jamadden commented:
> tox should be able to take advantage of pip caching, AFAIU -- unless the pip/virtualenv versions in the Travis image are too old?

Last time I checked they generally were.

> (The best reason I know not to use tox is when you want to use coverage run + coveralls.)

Also a good point!

jamadden commented:
I updated .travis.yml to fix all the tests and to enable coveralls. (Unfortunately coveralls is experiencing some issues right now.)

mgedmin commented on .travis.yml:
```diff
@@ -1,13 +1,25 @@
+cache: pip
+
+before_cache:
+    - rm -f $HOME/.cache/pip/log/debug.log
```
Now everything has 2-space indents except this line ;)

jamadden replied:
Fixed.

mgedmin reviewed (approved):
LGTM

mgedmin commented:
You have the PyPI bit for zope.app.schema now.

jamadden committed:
100% coverage.

Through 'no cover' in the package __init__.py files, and one on an
unexpected import error.

Also adjust the spacing in one line in .travis.yml

jamadden commented:
Thank you.
